client/post-editor/editor-sidebar/index.jsx

```diff
@@ -41,38 +55,47 @@ export default class EditorSidebar extends Component {
 			setPostDate,
 			isPostPrivate,
 			confirmationSidebarStatus,
+			nestedSidebar,
+			toggleNestedSidebar,
 		} = this.props;
 
+		const sidebarClassNames = classNames(
+			'editor-sidebar',
+			{ 'is-nested-sidebar-focused': nestedSidebar !== NESTED_SIDEBAR_NONE }
```

Comment:
Tiniest nit: 
`!== NESTED_SIDEBAR_NONE` seems like a double negative, perhaps `=== NESTED_SIDEBAR_REVISIONS` would be more suited? Unless of course the explicit intention is 'anything other than NESTED_SIDEBAR_NONE'